docs/language.rst

```diff
@@ -1757,6 +1757,64 @@ The syntax for calling external call is the same as the external call, except fo
 that it must be done on a contract type variable. Any error in an external call can
 be handled with :ref:`try-catch`.
 
+Internal calls and externals calls
+___________________________________
+
+An internal function call is executed by the current contract. This
+is much more efficient than an external call, which requires the
+address of the contract to call, whose arguments must be *abi encoded* (also known
+as serialization). Then, the runtime must set up the VM for the called contract
+(the callee), decode the arguments, and encode return values. Lastly,
+the first contract (the caller) must decode return values.
+
+A method call done on a contract type will always be an external call.
+Note that ``this`` returns the current contract, so ``this.foo()`` will do an
+external call, which is much more expensive than ``foo()``.
+
+Passing accounts with external calls on Solana
+______________________________________________
+
+The Solana runtime allows you the specify the accounts to be passed for an
+external call. This is specified in an array of the struct ``AccountMeta``, which
+has to be imported from the builtin file ``'solana'``. This import is builtin when
+you build your Solidity for Solana.
```

Comment:
It is a nice idea do add a section in the docs to talk about the contents of this 'solana' file.

Reply:
Specify the structs available, their attributes and the latter's types.

Reply:
I've introduced a builtin imports section.